utils/build/docker/java/spring-boot/src/main/java/com/datadoghq/system_tests/springboot/AppSecIastSource.java

```diff
@@ -104,4 +110,14 @@ private String find(final Collection<String> list,
         return find(list, matcher, Function.identity());
     }
 
+    @PostMapping("/multipart/test")
+    public String handleFileUpload(@RequestParam("file1") MultipartFile file) {
+        String fileContent = "NO_FILE";
+        try {
+            fileContent = Arrays.toString(file.getBytes());
+            file.getOriginalFilename();
```

Comment:
For the vulnerability to be trigger, you should use the filename in a sink, like SQL injection. See the other endpoints above.

Reply:
done